superchain/configs/sepolia/celo.toml

```diff
@@ -0,0 +1,67 @@
+sequencer_rpc = "https://forno.celo-sepolia.celo-testnet.org"
+explorer = "https://celo-sepolia.blockscout.com"
+superchain_level = 2
+governed_by_optimism = false
+data_availability_type = "alt-da"
+chain_id = 11142220
+batch_inbox_addr = "0x00076c1f80CdB1D670046465c5ccD04E9E78c1F0"
+block_time = 1
+seq_window_size = 7200
+max_sequencer_drift = 600
+gas_paying_token = "0x3C7011fD5e6Aed460cAa4985cF8d8Caba435b092"
+
+[hardforks]
+  canyon_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+  delta_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+  ecotone_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+  fjord_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+  granite_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+  holocene_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+  isthmus_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+
+[optimism]
+  eip1559_elasticity = 5
+  eip1559_denominator = 400
+  eip1559_denominator_canyon = 250
+
+[genesis]
+  l2_time = 1753283988
+  [genesis.l1]
+    hash = "0xeed417004236aa692023c216e8f1bd2185ac87a70c6f4393099f9c3777f05675"
+    number = 8825790
+  [genesis.l2]
+    hash = "0xabcc77fb92a7ea1e7d330042d81200585d251355b33408ed1c27fb54175b1fe2"
+    number = 0
+  [genesis.system_config]
+    batcherAddress = "0xE80D3F16E4D44A5127E1BD6ad9ed4E17A96475F0"
+    overhead = "0x0000000000000000000000000000000000000000000000000000000000000000"
+    scalar = "0x0100000000000000000000000000000000000000000000000000000000000000"
+    gasLimit = 60000000
+
+[roles]
+  SystemConfigOwner = "0x5e60d897Cd62588291656b54655e98ee73f0aabF"
+  ProxyAdminOwner = "0x5e60d897Cd62588291656b54655e98ee73f0aabF"
+  Guardian = "0x5e60d897Cd62588291656b54655e98ee73f0aabF"
+  Challenger = "0xC813b28614BD4CFA3d5Fdf153df41B273AB9D497"
+  Proposer = "0x640f7065C87c907C201B04d22f281D0586d4a85f"
+  UnsafeBlockSigner = "0xc01f0A060848058140296B0395663c67fDbbDbc4"
+  BatchSubmitter = "0xE80D3F16E4D44A5127E1BD6ad9ed4E17A96475F0"
+
+[addresses]
+  AddressManager = "0x8f0c6FC85A53551d87899aC2a5Af2B48C793eB63"
+  L1CrossDomainMessengerProxy = "0x70B0e58E6039831954eDE2EA1e9EF8a51680E4fD"
+  L1ERC721BridgeProxy = "0xB8c8dCBCCd0f7C5e7a2184b13B85D461d8711e96"
+  L1StandardBridgeProxy = "0xEc18a3c30131A0Db4246e785355fBc16E2eAF408"
+  L2OutputOracleProxy = "0x0000000000000000000000000000000000000000"
+  OptimismMintableERC20FactoryProxy = "0x261BE2eD7241feD9c746e0B5DFf3A4a335991377"
+  OptimismPortalProxy = "0x44AE3D41a335a7d05EB533029917aAd35662dcC2"
+  SystemConfigProxy = "0x760a5F022C9940f4A074e0030be682F560d29818"
+  ProxyAdmin = "0xF7d7A3d3bb8aBb6829249B3D3aD3d525D052027e"
+  SuperchainConfig = "0x31bEef32135c90AE8E56Fb071B3587de289Aaf77"
+  AnchorStateRegistryProxy = "0xD73BA8168A61F3E917F0930D5C0401aA47e269D6"
+  DelayedWETHProxy = "0x082F5f58B664CD1d51F9845fEE322aBA2cED9CbA"
+  DisputeGameFactoryProxy = "0x57C45d82D1a995F1e135B8D7EDc0a6BB5211cfAA"
+  MIPS = "0x0a691eEd7bE53F27f3C3b796061Cdb8565dA0b2a"
+  PreimageOracle = "0x855828eA44a0CE2596FDf49bEA5b2859c0453704"
```

Comment:
Did you manually create this file? Check some of the other chain config .toml files. They should now have a reduced address set stored here. I expect our automated chain config creation code to only write the subset of addresses to this file

Reply:
I just fixed the existing one and rebased on `main` - I will regenerate instead